src/utils.ts

```diff
@@ -47,7 +47,7 @@ const MAX_OUTPUT_SIZE = 1048576
 const MAX_CONTENT_SIZE = 10485760
 const ALLOWED_TYPES = ['image/jpeg', 'image/png']
 
-export async function download(ctx: Context, url: string, headers = {}): Promise<[ArrayBuffer, string]> {
+export async function download(ctx: Context, url: string, headers = {}): Promise<[buffer: ArrayBuffer, base64: string, dataUrl: string]> {
```

Comment:
This type is a bit complicated, it is recommended to return an interface `ImageData` instead.